src/ert/ensemble_evaluator/_ensemble.py

```diff
@@ -125,6 +120,9 @@ def __post_init__(self) -> None:
         else:
             self._status_tracker = _EnsembleStateTracker()
 
+        self.scheduler_queue: Any = None
+        self.manifest_queue: Any = None
```

Comment:
Do we need two queues, or can we have a general one and specify cloudevent types instead?

Reply:
We need to two queues yes, but can update to use `Queue[CloudEvent]` instead :+1: